### PR TITLE
fix: Made modulo more parsimonious. Changed return type from float to int

### LINKED ISSUE #2: Modulo function is inefficient
# `utils.py`
## `modulo`

line 60  features
`result = a - (np.floor(a / b) * b)`

which is an unnecessarily complicated way to calculate the modulo. 

Suggesting replacement with python's built-in modulo. `%`

### PR DESCRIPTION
**Pull request by denizmgun**
# Description
Python has a built-in modulo function- no need to reinvent the wheel

- resolves #2 

# Type of change

- **refactor**: A code change that neither fixes a bug nor adds a feature
- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)

# Questions (Optional)
- Not sure if you want to take int or float arguments. Arguments are int, but perhaps you want to run the modulo with floats as well?

# Remarks (Optional)
- See Question

### CODE REVIEW

**Comment by denizmgun** [comment]
@Maroali990 please review my code I'm afraid it is so much ugly :'(

**Review by A-Monti — APPROVED** [review]
Change generated is cleaner and easier to read.

File: src/utils.py [hunk 1]
```diff
@@ -52,13 +52,10 @@ def modulo(a: int, b: int):
...
     Returns:
-    float
+    int
     '''
 
-    # I think this could be made more efficient?
-    result = a - (np.floor(a / b) * b)
-
-    return result
+    return a % b
```
Comment (Maroali990) [inline src/utils.py:58]: Approved :D

**Review by Maroali990 — APPROVED** [review]
Looks good

**Pull request merged**